sdk/go/common/workspace/plugins_install_test.go

```diff
@@ -119,6 +118,9 @@ func assertPluginInstalled(t *testing.T, dir string, plugin PluginSpec) PluginIn
 	plugins, err := getPlugins(dir, skipMetadata)
 	assert.NoError(t, err)
 	assert.Equal(t, 1, len(plugins))
+	if len(plugins) == 0 {
+		assert.FailNow(t, "No plugins found")
```

Comment:
I has ```assert.Equal(t, 1, len(plugins))``` right above. Idomatic her would be:

```require.Equal(t, 1, len(plugins))```

The require part stops the test right away as FailNow.

Reply:
Ah yes, I forgot about the require assertion library. Thank you.